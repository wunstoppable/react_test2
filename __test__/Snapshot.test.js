/**
 * Created by WuTong on 2017/7/20.
 */
import React from 'react';
import AddTodoView from '../components/AddToDoView'
import renderer from 'react-test-renderer';


test('Link changes the class when hovered', () => {
    const component = renderer.create(
        <AddTodoView/>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})