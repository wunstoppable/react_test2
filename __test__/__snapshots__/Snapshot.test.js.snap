// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<header
  className="header"
>
  <h1>
    todos
  </h1>
  <input
    className="new-todo"
    onKeyUp={[Function]}
    placeholder="input todo item"
    type="text"
  />
</header>
`;
